### ISSUE TITLE
After updating from 0.13.0 to 0.14.0 sbt fails with Reference to undefined setting

### ISSUE THREAD

**Issue by barambani** [sbilinski/sbt-ecr#38]
After I bump the version from `0.13.0` to `0.14.0` the plugin fails at sbt load. The error is
```
[error] Reference to undefined setting:
[error]
[error]   app / Ecr / repositoryLifecyclePolicyText from app / Ecr / createRepository ((sbtecr.EcrPlugin.tasks) EcrPlugin.scala:43)
[error]
[error] 	at sbt.internal.util.Init.Uninitialized(Settings.scala:360)
[error] 	at sbt.internal.util.Init.Uninitialized$(Settings.scala:347)
[error] 	at sbt.Def$.Uninitialized(Def.scala:20)
[error] 	at sbt.internal.util.Init.delegate(Settings.scala:260)
[error] 	at sbt.internal.util.Init.delegate$(Settings.scala:236)
[error] 	at sbt.Def$.delegate(Def.scala:20)
[error] 	at sbt.internal.util.Init.compiled(Settings.scala:194)
[error] 	at sbt.internal.util.Init.compiled$(Settings.scala:182)
[error] 	at sbt.Def$.compiled(Def.scala:20)
[...]
```

**Comment by sbilinski** [comment]
@barambani 

Thanks for reporting this and sorry about the late reply! 

The fix should be present in `0.14.1` - the default setting for `repositoryLifecyclePolicyText` was missing apparently. 

-- 
cc: @mikegirkin

**Comment by barambani** [comment]
@sbilinski thank you for the fix, and for this very helpful plugin 👍 .
